src/videojs-hls.js

```diff
@@ -279,6 +291,11 @@ videojs.Hls.prototype.play = function() {
     this.mediaIndex = 0;
   }
 
+  if (this.playlists.media() && !this.el().classList.contains('vjs-has-started')) {
```

Comment:
We don't support IE8 in this project but you should get into the habit of using `this.hasClass()` for maximum compatibility.